internal/api/operations/include_test.go

```diff
@@ -69,13 +71,10 @@ func addIncludeTestData(appDB db.DB) {
 	tx.Connect(u1.ID(), pr.ID(), db.UserProfile.ID())
 
 	tx.Commit()
-}
+	up, _ := u1.Interface().RelationshipByName("posts")
```

Comment:
I'd prefer you use the literals (and make them work if they don't)

Reply:
(per http://www.haskellforall.com/2020/07/the-golden-rule-of-software-quality.html)

Reply:
So it was far from obvious to get the literals to work here (cause of circular reference issues). Happy to talk through this in the afternoon, but I actually think we should avoid having the literal representation outside of the db

Reply:
Maybe a comprimise is to create a db.UserPosts(tx) or something that returns the record version?